src/Pages/Tvshows/Tvshowid_Int.elm

```diff
@@ -56,16 +58,40 @@ init shared { params } =
     ( { tvshowid = params.tvshowid, tvshow = getTvShow params.tvshowid shared.tvshow_list, tvshow_list = shared.tvshow_list, season_list = shared.season_list, episode_list = shared.episode_list }
```

Comment:
We should reset the values here, if we know we're fetching new ones.

Reply:
These functions should take care of resetting the values when new ones are fetched.
```
save : Model -> Shared.Model -> Shared.Model
save model shared =
    { shared | season_list = model.season_list, episode_list = model.episode_list }


load : Shared.Model -> Model -> ( Model, Cmd Msg )
load shared model =
    ( { model | season_list = shared.season_list, episode_list = shared.episode_list }, Cmd.none )
```

Reply:
That's the thing, they don't. At least not in the current state, likely due to the Bug. I end up with Shows having seasons/episodes from other shows.

So a clean reset would at least hide that.

Reply:
Hi @razzeee,
I guess I know what could be the issue. The same thing was happening earlier before I fixed `streamdetails` where TvShows were having seasons/episodes from other shows due to irregularity in json data we received about a particular episode which led to `episode_list` not updating when the` tvshowid_int `page was opened. Let's say you have a `TvShow “X”` whose episode_list was successfully fetched, and the shared.episode_list was updated. When you went back tvshows main page and clicked to open another tvshow, the json data for this particular show had some irregularity which resulted in the `episode_list` not being able to fetch successfully, thus the `episode_list` didn’t get updated and still has the episodes from `TvShow ‘X’`.

It would be great if you could provide me with some of the TvShows that you are using in order to reproduce the issue so that I can compare and find the bug.

Reply:
I can probably send by database. Maybe you have more luck, i've been comparing a bunch.

Reply:
Okay sure, sounds good. I will look into it.

Reply:
Hi @razzeee,
I have made the required change for a clean reset.